File: benchmarks/json/generate_10k.js
const items = Array.from({length: 10000}, (_, i) => ({id: i, name: `item_${i}`, tags: ['t0','t1','t2','t3','t4'], value: i * 2.718}));
const start = Date.now();
const result = JSON.stringify(items);
console.log((Date.now() - start) * 1000);

File: benchmarks/json/generate_10k.rb
require 'json'
items = Array.new(10000) { |i| {id: i, name: "item_#{i}", tags: %w[t0 t1 t2 t3 t4], value: i * 2.718} }
start = Process.clock_gettime(Process::CLOCK_MONOTONIC, :microsecond)
result = JSON.generate(items)
finish = Process.clock_gettime(Process::CLOCK_MONOTONIC, :microsecond)
puts(finish - start)
